orttraining/orttraining/training_api/include/onnxruntime_training_c_api.h

```diff
@@ -0,0 +1,41 @@
+// This file contains c apis for on device training
+// This file should never be included standalone
+// It is included from within core/session/onnxruntime_c_api.h when
+// on device training is enabled
+// These apis can be moved to core/session/onnxruntime_c_api.h once they stabilize
+
+// DO NOT UNCOMMENT
+//#include "core/session/onnxruntime_c_api.h"
```

Comment:
Why do we need this comment?

Reply:
yes... I intentionally added it to make it clear that this is a part of c_api.h and it should not include the c_api.h Read the entire comment for more context